===================================================================
RCS file: /cvs/gnome/file-roller/src/eggtreemultidnd.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- eggtreemultidnd.c	2005/12/10 11:19:48	1.4
+++ eggtreemultidnd.c	2006/04/04 12:35:32	1.5
@@ -311,7 +311,7 @@
 
 	  context = gtk_drag_begin (widget,
 				    target_list,
-				    GDK_ACTION_COPY,
+				    GDK_ACTION_MOVE,
 				    priv_data->pressed_button,
 				    (GdkEvent*)event);
 	  set_context_data (context, path_list);
